### ISSUE TITLE
Unable to activate environment on Windows

### ISSUE THREAD

**Issue by necaris** [necaris/conda.el#68]
See https://github.com/necaris/conda.el/issues/47#issuecomment-674576255 from @notuntoward: unable to activate an existing environment using `(conda-env-activate "<name>")` in `.emacs`, despite it showing up in the list when running `M-x conda-env-activate`.

**Comment by arthurcgusmao** [comment]
Context: @notuntoward is experiencing the following error, on Windows,
```text
Error (use-package): conda/:config: Invalid conda environment path
specified: c:/Users/scott/anaconda3/envs/stdso/
```
when trying to run
```lisp
(conda-env-activate "stdso")
```

That error is raised by the `conda-env-activate-path` function when the call
```lisp
(conda-env-is-valid-path "c:/Users/scott/anaconda3/envs/stdso/")
```
returns nil.

@notuntoward, can you please check that:

1. The value of `conda-env-executables-dir` is "Scripts";
2. The file `c:/Users/scott/anaconda3/envs/stdso/Scripts/python.exe` exists;
3. The output of both
```lisp
(concat (expand-file-name "c:/Users/scott/anaconda3/envs/stdso/")
				    (file-name-as-directory conda-env-executables-dir)
				    (if (eq system-type 'windows-nt) "python.exe" "python"))
```
and
```lisp
(concat (file-name-as-directory (expand-file-name "c:/Users/scott/anaconda3/envs/stdso/"))
				    (file-name-as-directory conda-env-executables-dir)
				    (if (eq system-type 'windows-nt) "python.exe" "python"))
```
and let us know the results?

**Comment by notuntoward** [comment]
@arthurcgusmao wrote: can you please check that:

1. The value of conda-env-executables-dir is "Scripts";
**Answer: Yes**

2. The file c:/Users/scott/anaconda3/envs/stdso/Scripts/python.exe exists;
**Answer: No**  This must be the problem.  python.exe is in:
 c:/Users/scott/anaconda3/envs/stdso/python.exe 
  
3. The output of both

**output of 1st command:** 
"c:/Users/scott/anaconda3/envs/stdso/Scripts/python.exe"

**output of 2nd command:** 
"c:/Users/scott/anaconda3/envs/stdso/Scripts/python.exe"

**Comment by arthurcgusmao** [comment]
@notuntoward Yes, probably. That function was introduced in 2dce0ac, but it was only being used by the `conda-env-activate-path` function. My refactoring made `conda-env-activate` use the former as a subroutine, which revealed the bug.

I have created a fix that should solve the issue: https://github.com/arthurcgusmao/conda.el/tree/fix-68-activate-Windows

Can you please git it a try? An easy way to do it is to just evaluate the new file `conda.el` and try activating an environment.

**Comment by necaris** [comment]
@notuntoward thank you for the detailed bug report! I hope that @arthurcgusmao's helpful fix in #71 has solved this -- please let us know if that's not the case!

**Comment by notuntoward** [comment]
@arthurcgusmao and @necaris,   Thanks -- both problems are fixed.
